doc/Interns/danbugs/MyFirstCommit.md

```diff
@@ -0,0 +1 @@
+May 5, 2020
```

Comment:
Sorry, one more change.  Please add the time to this file.

Reply:
Done! ✅